flink-connectors/flink-sql-connector-elasticsearch7/src/main/resources/META-INF/NOTICE

```diff
@@ -6,10 +6,12 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
+- com.carrotsearch:hppc:0.8.1
```

Comment:
hppc has a NOTICE file that we need to forward: https://github.com/carrotsearch/hppc/blob/0.8.1/NOTICE.txt

I could not find it in this archive: https://repository.apache.org/content/repositories/orgapacheflink-1362/org/apache/flink/flink-sql-connector-elasticsearch7_2.11/1.10.1/flink-sql-connector-elasticsearch7_2.11-1.10.1.jar

Reply:
Since the jar does not contain the NOTICE we do not have to forward it imo.

> (d) If the Work includes a "NOTICE" text file as part of its
>           **distribution**, then any Derivative Works that You distribute must
>           include a readable copy of the attribution notices contained
>           within such NOTICE file

Reply:
Thanks a lot for this pointer! I learned something new about licenses tonight.

I rest my case.